### PR TITLE
Change notification buttons to variables

### PR DESCRIPTION
**Pull request by peterlewis**
Changes button text from static to variable-defined (stored in .plist)

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -106,6 +112,8 @@ defaults write "$PWD/$onDemainDaemonFileName" Label -string "$onDemandDaemonIden
 defaults write "$PWD/$preferenceFileName" UpdateInProgressTitle -string "$updateInProgressTitle"
 defaults write "$PWD/$preferenceFileName" UpdateInProgressMessage -string "$updateInProgressMessage"
 defaults write "$PWD/$preferenceFileName" LoginAfterUpdatesInProgressMessage -string "$loginAfterUpdatesInProgressMessage"
+defaults write "$PWD/$preferenceFileName" notificationActionButtonText -string "$notificationActionButtonText"
```
Comment (ryangball) [inline build.sh]: Change the key that is being written to NotificationActionButtonText (uppercase N) to stay consistent with the other keys in the plist. The variable can keep the lowercase n.

File: build.sh [hunk 2]
```diff
@@ -106,6 +112,8 @@ defaults write "$PWD/$onDemainDaemonFileName" Label -string "$onDemandDaemonIden
...
+defaults write "$PWD/$preferenceFileName" notificationOtherButtonText -string "$notificationOtherButtonText"
```
Comment (ryangball) [inline build.sh]: Change the key that is being written to NotificationOtherButtonText (uppercase N) to stay consistent with the other keys in the plist. The variable can keep the lowercase n.

File: nice_updater.sh [hunk 3]
```diff
@@ -13,6 +13,8 @@ preferenceFileFullPath="/Library/Preferences/com.github.ryangball.nice_updater.p
 updateInProgressTitle=$(defaults read "$preferenceFileFullPath" UpdateInProgressTitle)
 updateInProgressMessage=$(defaults read "$preferenceFileFullPath" UpdateInProgressMessage)
 loginAfterUpdatesInProgressMessage=$(defaults read "$preferenceFileFullPath" LoginAfterUpdatesInProgressMessage)
+notificationActionButtonText=$(defaults read "$preferenceFileFullPath" notificationActionButtonText)
```
Comment (ryangball) [inline nice_updater.sh]: Change the key that is being read to NotificationActionButtonText (uppercase N) to stay consistent with the other keys in the plist. The variable can keep the lowercase n.

File: nice_updater.sh [hunk 4]
```diff
@@ -13,6 +13,8 @@ preferenceFileFullPath="/Library/Preferences/com.github.ryangball.nice_updater.p
...
+notificationOtherButtonText=$(defaults read "$preferenceFileFullPath" notificationOtherButtonText)
```
Comment (ryangball) [inline nice_updater.sh]: Change the key that is being written to NotificationOtherButtonText (uppercase N) to stay consistent with the other keys in the plist. The variable can keep the lowercase n.

**Review by ryangball — CHANGES_REQUESTED** [review]
Not trying to nit-pick, but this is just to keep the plist keys all consistent beginning with uppercase letters. All the variable names can stay how they are. Thank you ahead of time.

File: build.sh (hunk 2 above)
Comment (peterlewis) [reply build.sh]: Makes complete sense. Sorry for not spotting it! :)

File: build.sh (hunk 1 above)
Comment (peterlewis) [reply build.sh]: Makes complete sense. Sorry for not spotting it! :)

File: nice_updater.sh (hunk 4 above)
Comment (peterlewis) [reply nice_updater.sh]: Makes complete sense. Sorry for not spotting it! :)

File: nice_updater.sh (hunk 3 above)
Comment (peterlewis) [reply nice_updater.sh]: Makes complete sense. Sorry for not spotting it! :)

**Comment by peterlewis** [comment]
Fret not, no nit-picking detected! It needed to be fixed. I've now made those changes.

**Pull request merged**